web_poet/mixins.py

```diff
@@ -5,7 +5,17 @@
 from w3lib.html import get_base_url
 
 
-class SelectableMixin(abc.ABC):
+class CssXpathMixin:
```

Comment:
Could we go for a more generic name? (I’m thinking of [JMESPath support in the future](https://github.com/scrapy/parsel/pull/181)) e.g. `SelectorShortcutsMixin` would be in line with `ResponseShortcutsMixin`.